web/src/App.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Switch, Route, HashRouter } from 'react-router-dom';
+import Layout from './components/layout';
+import Home from './containers/home';
+import Login from './containers/login';
+import EditPoll from './containers/editPoll';
+import Register from './containers/register';
+
+function App(): JSX.Element {
+
+  return (
+    <HashRouter>
```

Comment:
I see `BrowserRouter` used web/src/index.tsx

Why do we need `HashRouter` here?